### ISSUE TITLE
[202012] warm-reboot from 201811 to 202012 with one LAG down results in recreation of multiple LAG_MEMBER objects during APPLY_VIEW

### ISSUE THREAD

**Issue by tjchadaga** [Azure/sonic-buildimage#9767]
#### Description

Issue was seen when warm-reboot was done on a system with 4 LAGs, where one LAG is admin down before warm-reboot. After warm-reboot, it is seen that  along with the LAG_MEMBER that is down,  two or three other LAG_MEMBER objects get removed and created during APPLY_VIEW. This also results in recreation of other objects dependent on these LAG_MEMBER objects like ROUTER_INTERFACE ->NEXT_HOPS -> NEIGHBOR_ENTRY

#### Steps to reproduce the issue:
1. Load 201811 image and configure 3 or more LAGs
2. Admin shut one LAG
3. warm-reboot to 202012

#### Describe the results you received:
Observed 3 out of 4 LAG_MEMBER objects recreated during APPLY_VIEW

#### Describe the results you expected:
Expected only 1 LAG_MEMBER create operation during warm-reboot (corresponding to the shut LAG). This is expected due to the difference in behavior between 201811 (admin down LAG has no lag members) vs 202012 (admin down LAG still has members)

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by vaibhavhd** [comment]
Attaching logs from KVM - this contains INFO level logs, SAIdump (on 201811, 202012) and techsupport:

[logs-lagdown-upgrade.zip](https://github.com/Azure/sonic-buildimage/files/7872208/logs-lagdown-upgrade.zip)

In this example, `PortChannel102` was shutdown on DUT running 201811 image, and then warmboot to 202012 was done.
```
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_ACL_ENTRY: 2
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_ACL_TABLE: 5
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_ACL_TABLE_GROUP_MEMBER: 64
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_BRIDGE_PORT: 24
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_HOSTIF_TRAP: 11
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_HOSTIF_TRAP_GROUP: 6
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_LAG_MEMBER: 3  <========================
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_NEIGHBOR_ENTRY: 4
Jan 14 16:24:39.699863 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_NEXT_HOP: 4
Jan 14 16:24:39.699883 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER: 3
Jan 14 16:24:39.699891 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_POLICER: 3
Jan 14 16:24:39.699899 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_ROUTER_INTERFACE: 3
Jan 14 16:24:39.699899 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_ROUTE_ENTRY: 7
Jan 14 16:24:39.699913 vlab-01 NOTICE syncd#syncd: :- executeOperationsOnAsic: operations on SAI_OBJECT_TYPE_SWITCH: 2

Jan 14 16:24:39.699913 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: index update from 4 to 5
Jan 14 16:24:39.699924 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move 0x120000000005b6 in the middle up: SAI_OBJECT_TYPE_POLICER (last: 5 curr: 5)
Jan 14 16:24:39.699924 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x1b00000000065a all way up (not in map): SAI_OBJECT_TYPE_LAG_MEMBER to index: 6  <=====================
Jan 14 16:24:39.699934 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005b9 all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 7
Jan 14 16:24:39.699943 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005ba all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 8
Jan 14 16:24:39.699943 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005bb all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 9
Jan 14 16:24:39.699955 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005bd all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 10
Jan 14 16:24:39.699955 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005be all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 11
Jan 14 16:24:39.699967 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005bf all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 12
Jan 14 16:24:39.699967 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005c2 all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 13
Jan 14 16:24:39.699981 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005c4 all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 14
Jan 14 16:24:39.699981 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005c5 all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 15
Jan 14 16:24:39.699993 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x220000000005c6 all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 16
Jan 14 16:24:39.699993 vlab-01 INFO syncd#syncd: :- asicGetWithOptimizedRemoveOperations: move oid:0x22000000000616 all way up (not in map): SAI_OBJECT_TYPE_HOSTIF_TRAP to index: 17

```

**Comment by kcudnik** [comment]
thanks for explanation, i will be working on this to figure out whether there is a way to correctly match LAG/LAG_MEMBER and not cause remapping, i already have logs for this scenario thanks to [vaibhavhd](https://github.com/vaibhavhd), i need to do a repro locally and then will try to fix this

**Comment by tjchadaga** [comment]
> Fixed here: [Azure/sonic-sairedis#1008](https://github.com/Azure/sonic-sairedis/pull/1008) can you cherry pick and test this if this solves the issue?

Issue is not seen after porting this PR to 202012.
